support-frontend/assets/components/thankYou/signUp/signUpItems.tsx

```diff
@@ -17,30 +20,43 @@ export function SignUpBodyCopy({
 		'Make sure you sign in on each of the devices you use to read our journalism – either today or next time you use them.';
 	const lowerCopyTier3 =
 		'Make sure you sign in on all your devices when browsing our website and app. This helps us recognise you as a valued subscriber so you can enjoy all the benefits included in your subscription.';
+	const observerCopy =
+		'To finish creating your account, please check your inbox for an email from us. This step will complete your account setup and will allow you to manage your subscription.';
+
 	return (
 		<>
-			<p>{isTier3 ? upperCopyTier3 : upperCopy}</p>
-			{!isTier3 && (
-				<div
-					css={css`
-						margin-top: ${space[4]}px;
-					`}
-				>
-					<BulletPointedList
-						items={[
-							'Show you far fewer requests for financial support',
-							'Offer you a simple way to manage your support payments and newsletter subscriptions',
-						]}
-					/>
-				</div>
-			)}
-			<p
-				css={css`
-					margin-top: ${space[4]}px;
-				`}
-			>
-				{isTier3 ? lowerCopyTier3 : lowerCopy}
+			<p>
+				{observerPrint
+					? observerCopy
+					: isTierThree
+					? upperCopyTier3
+					: upperCopy}
 			</p>
+			{!observerPrint && (
+				<>
+					{!isTierThree && (
```

Comment:
could we not bundle these together? 
```suggestion
			{!observerPrint &&  !isTierThree && (
```

Reply:
@maloky  here is a `<p>` further down that is not-observer-only that would be hidden if I combine I think?

Reply:
ahh, missed that, you right 👍